reporting-app/app/views/dashboard/index.html.erb

```diff
@@ -1,13 +1,19 @@
-<h1><%= t(".title") %></h1>
+<div class="grid-container">
+  <h1 class="margin-bottom-2"><%= t(".title") %></h1>
+</div>
 <% if @information_request %>
-  <%= render "dashboard/request_for_information", information_request: @information_request %>
+  <div class="grid-container">
+    <%= render "dashboard/request_for_information", information_request: @information_request %>
+  </div>
 <% end %>
 <%= render determine_dashboard_view, activity_report: @activity_report_application_form, exemption_application: @exemption_application_form, certification_case: @certification_case, certification: @certification %>
 <!-- Previous Certifications -->
 <% if @all_certifications.length > 1 %>
-  <h2><%= t(".previous_certifications.title") %></h2>
-  <p class="font-serif-md">
-    <%= t(".previous_certifications.intro") %>
-  </p>
-  <%= link_to t(".previous_certifications.review_previous_certifications_button"), certifications_path, class: "usa-button usa-button--outline" %>
+  <div class="grid-container">
```

Comment:
Adding this `div` removes the top margin from the pervious certifications header. 

This branch:

<img width="1152" height="477" alt="Image" src="https://github.com/user-attachments/assets/7a496e86-098d-40f1-b1db-ee122ccebc48" />


main:

<img width="1094" height="544" alt="Image" src="https://github.com/user-attachments/assets/bc72a9b8-5f34-4e72-abd8-595fe5e6e7b3" />

you could add a USWDS margin class here

```suggestion
  <div class="grid-container margin-top-4">
```